api/v1alpha1/condition_consts.go

```diff
@@ -9,15 +9,34 @@ const (
 	ReadyCondition ConditionType = "Ready"
 )
 
+// Conditions and condition Reasons for the VirtualMachine object.
+
+const (
+	// VirtualMachinePrereqReadyCondition documents that all of a VirtualMachine's prerequisites declared in the spec
+	// (e.g. VirtualMachineClass) are satisfied.
+	VirtualMachinePrereqReadyCondition ConditionType = "VirtualMachinePrereqReady"
+
+	// VirtualMachinePrereqReadyReason documents that all of a VirtualMachine's PreRequisites have been satisfied.
```

Comment:
Do we need to set a Reason while the condition is true?

Reply:
No.

https://github.com/kubernetes-sigs/cluster-api/blob/dadd7bd6233edb1e422e7af282e923abff52f830/util/conditions/setter.go#L109

Reply:
In that case do we need VirtualMachinePrereqReadyReason?

Reply:
Removed the `Reason` for the True condition. Thanks for pointing that out.